components/SatisfactionSlider.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react'
+
+import styles from '../styles/SatisfactionSlider.module.css'
+
+import Smiley from './Smiley'
+
+type Props = {
+  initial?: number
+  max?: number
+  min?: number
+  step?: number
+  title?: string
+  updateCallback?: (update: number) => void
+}
+const SatisfactionSlider = ({
+  initial = 6,
+  max = 10,
+  min = 0,
+  step,
+  title,
+  updateCallback
+}: Props) => {
+  const [number, updateNumber] = useState(initial || 0)
+
+  useEffect(
+    () => {
+      if (!updateCallback) return
+      updateCallback(number)
+    },
+    // The eslint rule is correct in identifying that if you change the callback
+    // it may not get fired until the value is updated
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [number]
+  )
+
+  const scaledNumber = (number - min) / (max - min)
+
+  return (
+    <span className={styles.wrapper}>
+      {title && <h3>{title}</h3>}
```

Comment:
If there is only one question per screen then you can revert to `<h1>`?

Reply:
In case a user for some reason doesn't load our css I don't mind it being an `h3` since there may be a truly more important `h1` on the title screen